Slim/Control/Commands.pm

```diff
@@ -380,8 +380,9 @@ sub clientForgetCommand {
 	# Persist playback state like we would do when turning off a player, that is, treat a vanishing
 	# player that's still playing the same way as a player that's turned off while still playing, so
 	# we can make it start playing again if it reappears and the user told us to resume playing
-	# when powering on.
-	$client->persistPlaybackStateForPowerOff();
+	# when powering on. Make sure elapsed is not 0 if we are playing
+	my $playing = $client->controller->playingSongElapsed() + 0.1 if $client->controller->isPlaying(1);
```

Comment:
Why do we have to make sure elapsed time is not 0?

Reply:
Because 0 means we are not playing at power off